pkg/limits/flags.go

```diff
@@ -0,0 +1,123 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license
+
+package limits
+
+import (
+	"flag"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/timescale/promscale/pkg/limits/mem"
+	"github.com/timescale/promscale/pkg/util"
+)
+
+var (
+	MemoryTargetMetric = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: util.PromNamespace,
+			Name:      "memory_target_bytes",
```

Comment:
as per best practices: https://prometheus.io/docs/practices/naming/#metric-names

```suggestion
			Name:      "target_memory_usage_bytes",
```

Reply:
The word `target` is misleading. Something like `memory_usage_bytes` seems good to me., as the target system is the current system and current is an obvious word in this context, so it's not needed. Hence, memory_usage_bytes.

Reply:
This gauge is not the current usage it is the upper limit we are targeting

Reply:
I'm not against another name, but I don't think the one you suggest is better